ui/panel.js

```diff
@@ -0,0 +1,151 @@
+// -*- mode: js; js-indent-level: 4; indent-tabs-mode: nil -*-
```

Comment:
These headers aren't necessary anymore these days

Reply:
I really would prefer to not remove these as they are simply in every js file - if you insist I'd add a new commit removing it from all files at once but I wouldn't bother - btw, why it it not needed anymore?